README.md

```diff
@@ -11,7 +11,7 @@ They are also used for each release of the following partner projects:
 - [u-track](http://downloads.openmicroscopy.org/u-track)
 - [OMERO.webtagging](http://downloads.openmicroscopy.org/webtagging)
 - [OMERO.searcher](http://downloads.openmicroscopy.org/searcher)
-- [OMERO.figure](http://downloads.openmicroscopy.org/figure)
+- [OMERO.figure](http://downloads.openmicroscopy.org/figure) (legacy)
```

Comment:
Do we want to update http://downloads.openmicroscopy.org/ with the same note?  NB: `legacy` reads ambiguous to me as it might convey the idea the project is discontinued while effectively 1- it is promoted to core and 2- its deployment mechanism is migrated to PyPi